mmv1/third_party/terraform/services/monitoring/resource_monitoring_alert_policy_test.go

```diff
@@ -457,6 +473,10 @@ resource "google_monitoring_alert_policy" "promql" {
     content   = "test content"
     mime_type = "text/markdown"
     subject = "test subject"
+	links {
+		displayname = "link display name"
```

Comment:
```suggestion
		displayName = "link display name"
```

Reply:
> 

You suggested "displayName" but I think "display_name" should be the correct format according to other "display_name" fields. 
Can you confirm? Thank you!